Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "pdfium-render"
 version = "0.8.25"
 edition = "2018"
-rust-version = "1.60"
```

Comment:
This change breaks the published MSRV for this crate. You need to clearly and persuasively explain why you are doing this.